src/allotropy/allotrope/schema_parser/model_class_editor.py

```diff
@@ -307,6 +343,8 @@ def create(
 
         for field_name in fixed_field_name_order:
             lines.append(f"    {fields[field_name].contents}")
+        if not fixed_field_name_order:
+            lines.append("    pass")
```

Comment:
Nit: it seems like 4 spaces + string is a pretty common pattern. Util?

Reply:
So far just these two lines. I hope this will go away after schema generator revamp is done in the next month of so, so don't want to over-design here.